src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -145,6 +145,7 @@ const StreamActions = createReactClass({
     realtimeActive: PropTypes.bool.isRequired,
     statsPeriod: PropTypes.string.isRequired,
     query: PropTypes.string.isRequired,
+    environment: PropTypes.instanceOf(Object),
```

Comment:
We can do SentryTypes.Environment since it's defined https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/proptypes.jsx#L231